### ISSUE TITLE
Remove ChiSquared Distribution in favor of helper

### ISSUE THREAD

**Issue by ricardoV94** [pymc-devs/pymc#6994]
### Description

It's just a `Gamma(alpha=nu/2, beta=1/2)`. No point in having the extra code:

https://github.com/pymc-devs/pymc/blob/ec4407d3db2c8ebe77e62c65f2eda3f5703e6279/pymc/distributions/continuous.py#L2378-L2434

```python
class Chisquare:
  """docs"""

  def __new__(cls, name, nu, **kwargs):
    return Gamma(name, nu/2, 1/2, **kwargs)
  
  @classmethod
  def dist(cls, nu, **kwargs):
    return Gamma.dist(nu/2, 1/2, **kwargs)
```

Likewise, the `ChiSquareRV` in PyTensor can be removed in favor of a helper that just returns the right gamma:

https://github.com/pymc-devs/pytensor/blob/3169197c54e2cab7ba27c647c760b93e12abca5e/pytensor/tensor/random/basic.py#L490-L540

**Comment by wd60622** [comment]
I'll take this. I think I have it down 😃

**Comment by ricardoV94** [comment]
We should add a pymc-examples using CustomDist first. That would be much more impactful if you want to contribute

**Comment by wd60622** [comment]
> We should add a pymc-examples using CustomDist first. That would be much more impactful if you want to contribute

Sounds good. I will take a look at what exists and add something if needed

**Comment by ricardoV94** [comment]
Some nice examples that we have been meaning to illustrate involve simple timeseries: https://gist.github.com/ricardoV94/a49b2cc1cf0f32a5f6dc31d6856ccb63

**Comment by wd60622** [comment]
> Some nice examples that we have been meaning to illustrate involve simple timeseries: https://gist.github.com/ricardoV94/a49b2cc1cf0f32a5f6dc31d6856ccb63
> 
> We should however start with something simpler. The Maxwell distribution is a good example (as well as the two last examples in the docstrings).
> 
> This is a bit related to #6603

Sounds good. I see [this current one](https://www.pymc.io/projects/examples/en/latest/howto/custom_distribution.html) ([Notebook here](https://github.com/wd60622/pymc-examples/blob/main/examples/howto/custom_distribution.ipynb)) as the only one that touches this topic. Should I append to this or make something new? 

I like your time series example and could work up to that. I can note distributions like skellam (difference of two distribution), BetaNegativeBinomial (compound distribution), and maxwell (scaled distribution) as they all small examples and touch some differences. I like your probabilities puzzles too which I would like to link. 

I will open up an pymc-examples issue for this

**Comment by ricardoV94** [comment]
Closed via #7007

**Comment by ricardoV94** [comment]
Thanks @wd60622
